__tests__/text.test.ts

```diff
@@ -454,3 +456,25 @@ it("generateWordFromId", () => {
   expected = "FinTech"
   expect(actual).toBe(expected)
 })
+
+it("generateSlugFromWords", () => {
+  let actual = generateSlugFromWords("Hello world post by mo", "qdsad123asd12dasd")
+  let expected = "hello-world-post-by-mo-qdsad123asd12dasd"
+
+  expect(actual).toBe(expected)
+
+  actual = generateSlugFromWords("qdsad123asd12dasd")
+  expected = "qdsad123asd12dasd"
+  expect(actual).toBe(expected)
+})
+
+it("extractIDfromSlug", () => {
```

Comment:
add tests for empty string, null and undefined

Reply:
addressed